source/common/stats/symbol_table_impl.h

```diff
@@ -63,8 +74,26 @@ using SymbolVec = std::vector<Symbol>;
  * same string is re-encoded, it may or may not encode to the same underlying
  * symbol.
  */
-class SymbolTableImpl : public SymbolTable {
```

Comment:
Also change the names of files?

Also, if this class is not expected to be mocked, does it make sense declaring it as final?

Reply:
I added 'final' - though that's not something that happens much in Envoy C++ code for other classes that are not expected to be derived from.

I'm wondering if you think the compiler might be able to optimize better with this keyword in this case (no `virtual` anything), or whether this is just for the human reader. Either way I'm fine.

Reply:
It was more to validate that we indeed don't need an interface for the class, and this is the one and only implementation.